src/Microsoft.Extensions.Logging/Microsoft.Extensions.Logging.csproj

```diff
@@ -17,6 +17,8 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Configuration" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="$(AspNetCoreVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Options" Version="$(AspNetCoreVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="$(AspNetCoreVersion)" />
```

Comment:
I think the most important thing is for `M.E.L.Abstractions` to be clean because that's what libraries will depend on. `M.E.L` will be used by hosts and they have all of these dependencies anyway, so it's probably not necessary to split this?!